src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin trans ($select: all, $dur: #{$duration}, $type: linear) {
```

Comment:
transplantation, transport, or transition :) There could be a lot of options for such shorthand. So write full names to describe the functionality